### PR TITLE
Add user validation login page with  POM methods

### PR DESCRIPTION
**Pull request by Sanberqui18**
Tests Passed:
Screen Shot 2023-12-13 at 9 07 50 AM

### CODE REVIEW

File: cypress/e2e/user.validation.cy.ts [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+          loginPage.signIn(
+            users.standard_user.username,
+            users.standard_user.password,
+          );
+        });
+
+        productPage.displayContainer();
+      });
+
+      describe("When I try to log in using wrong credentials", () => {
```
Comment (salorrego) [inline cypress/e2e/user.validation.cy.ts]:
```suggestion
      describe("and I try to log in using wrong credentials", () => {
```

File: cypress/support/types.d.ts [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+// declare global {
+//   namespace Cypress {
+//     interface Chainable {
+//       fixture<K extends keyof FixtureTypes>(
+//         fixtureName: K,
+//       ): Chainable<FixtureTypes[K]>;
+//     }
+//   }
+// }
+
```
Comment (salorrego) [inline cypress/support/types.d.ts]: si no lo necesitas lo puedes borrar

**Review by salorrego — CHANGES_REQUESTED** [review]

File: cypress/e2e/user.validation.cy.ts (hunk 1 above)
Comment (Sanberqui18) [reply cypress/e2e/user.validation.cy.ts]: Done

File: cypress/support/types.d.ts (hunk 2 above)
Comment (Sanberqui18) [reply cypress/support/types.d.ts]: Done

File: cypress/e2e/user.validation.cy.ts (hunk 1 above)
Comment (salorrego) [reply cypress/e2e/user.validation.cy.ts]: not done

**Review by salorrego — CHANGES_REQUESTED** [review]

**Review by salorrego — APPROVED** [review]
ok

**Pull request merged**